sample/kotlin/src/main/kotlin/com/datadog/android/sample/picture/PictureViewModel.kt

```diff
@@ -40,7 +40,7 @@ internal class PictureViewModel : ViewModel() {
     }
 
     companion object {
-        private const val RANDOM_URL = "https://source.unsplash.com/random/800x450"
+        private const val RANDOM_URL = "https://imgix.datadoghq.com//img/about/presskit/Bits_Mascot.jpg"
```

Comment:
Why do we change it? Aren't we making all the tests for the images in new `ImageComponentsFragment.kt`?

Reply:
yes, I guess unsplash.com killed their API some time ago. I did the update for instrumented tests some time ago https://github.com/DataDog/dd-sdk-android/commit/5e95b641d4f1ca0aa0b45ff8ef74fba100566589, but didn't do it for the sample app. Can you please do a similar update for the `unsplash` links here, please? :)